http/src/main/resources/init-resources/jupyter-docker-compose.yaml

```diff
@@ -14,7 +14,7 @@ services:
     network_mode: host
     volumes:
       # shared with welder
-      - /work:/home/jupyter-user/notebooks # TODO: make this ${NOTEBOOKS_DIR} once welder is rolled out to all clusters
+      - /work:${NOTEBOOKS_DIR}
```

Comment:
Wondering if this should be quoted in order to use a `${..}` variable?

Like here: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml#L17

Reply:
probably not since both welder and jupyter can access the directory fine based on testing in fiab